libs/api/domains/rsk-company-info/src/lib/graphql/rsk-company-info.resolver.ts

```diff
@@ -0,0 +1,77 @@
+import { Inject, UseGuards } from '@nestjs/common'
+import { Args, Parent, Query, ResolveField, Resolver } from '@nestjs/graphql'
+import { RskCompany } from './models/rskCompany.model'
+import { Audit } from '@island.is/nest/audit'
+import { IdsUserGuard, ScopesGuard } from '@island.is/auth-nest-tools'
+import { RskCompanyInfoService } from './rsk-company-info.service'
+import { RskCompanyInfoInput } from './dto/RskCompanyInfo.input'
+import { RskCompanyInfoSearchInput } from './dto/RskCompanyInfoSearch.input'
+import { RskCompanySearchItems } from './models/rskCompanySearchItems.model'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+import type { Logger } from '@island.is/logging'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@Resolver(() => RskCompany)
+export class RskCompanyInfoResolver {
+  constructor(
+    private rskCompanyInfoService: RskCompanyInfoService,
+    @Inject(LOGGER_PROVIDER)
+    private logger: Logger,
+  ) {}
+
+  @Query(() => RskCompany, {
+    name: 'rskCompany',
+    nullable: true,
+  })
+  // @Audit()
```

Comment:
Not sure if we need audit in this domain (company registry lookups). @hrefnalind?